web-console/src/app.html

```diff
@@ -7,7 +7,7 @@
     %sveltekit.head%
   </head>
 
-  <body data-sveltekit-preload-data="hover" data-theme="feldera-theme" class="h-screen">
+  <body data-sveltekit-preload-data="hover" data-theme="feldera-modern-theme" class="h-screen">
```

Comment:
no need to keep the old theme around

Reply:
We don't **need** to, yes
On a serious note, it may be useful atleast temporarily - as a baseline to compare against to make sure e.g. the colors work out

Reply:
we can always use v0.23 to compare